.github/workflows/pull-request.yml

```
@@ -7,39 +7,30 @@ on:
+        strategy:
+          matrix:
+            os: [ubuntu-latest, macos-latest]
+            swift: ["5.10", "6.0", "6.1"]
+        runs-on: ${{ matrix.os }}
         steps:
-            - uses: actions/checkout@v4
-            - name: Build (macOS)
-              run: swift build -v
-            - name: Run tests
-              run: swift test -v
+        - uses: swift-actions/setup-swift
```

actionlint (1.7.7)

specifying action "swift-actions/setup-swift" in invalid format because ref is missing. available formats are "{owner}/{repo}@{ref}" or "{owner}/{repo}/{path}@{ref}"

(action)